packages/conventional-recommended-bump/cli.js

```diff
@@ -1,55 +1,57 @@
 #!/usr/bin/env node
+
 'use strict';
-var meow = require('meow');
-var conventionalRecommendedBump = require('./');
-var path = require('path');
 
-var cli = meow({
+const meow = require(`meow`);
+const conventionalRecommendedBump = require(`./`);
+const path = require(`path`);
+
+const cli = meow({
   help: [
-    'Usage',
-    '  conventional-recommended-bump',
-    '',
-    'Example',
-    '  conventional-recommended-bump',
-    '',
-    'Options',
-    '  -p, --preset                   Name of the preset you want to use',
-    '  -g, --config                   A filepath of your config script',
-    '  -h, --header-pattern           Regex to match header pattern',
-    '  -c, --header-correspondence    Comma separated parts used to define what capturing group of `headerPattern` captures what',
-    '  -r, --reference-actions        Comma separated keywords that used to reference issues',
-    '  -i, --issue-prefixes           Comma separated prefixes of an issue',
-    '  -n, --note-keywords            Comma separated keywords for important notes',
-    '  -f, --field-pattern            Regex to match other fields',
-    '  -v, --verbose                  Verbose output',
-    '  -l, --lerna-package            Recommend a bump for a specific lerna package (:pkg-name@1.0.0)',
-    '  -t, --tag-prefix               Tag prefix to consider when reading the tags',
-    '  --commit-path                  Recommend a bump scoped to a specific directory'
+    `Usage`,
```

Comment:
with template literals, we don't need to use an array.

Reply:
Fixed.